content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan sebuah gambaran umum dari `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa *secret*, *volume*, *volume mount*, dan variabel *environment*.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami *Pod Preset*
+---
+
+Sebuah *Pod Preset* adalah sebuah *resource* API untuk memasukkan kebutuhan *runtime* tambahan ke dalam sebuah *Pod* pada saat waktu penciptaan. Kamu akan menggunakan *label selector* untuk menunjuk *Pod-pod* dimana *Pod Preset* diterapkan.
+
+Menggunakan sebuah *Pod Preset* memungkinkan pembuat templat *pod* untuk tidak menyediakan secara eksplisit semua informasi untuk setiap *pod*. Dengan demikian, pembuat templat *pod* mengkonsumsi sebuah *service* spesifik untuk tidak perlu tahu semua detail-detail tentang *service* tersebut.
+
+Untuk informasi lebih lanjut mengenai latar belakang lihat [proposal desain untuk *PodPreset*](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/service-catalog/pod-preset.md).
+
+## Bagaimana Cara Kerja *Pod Preset*
+---
+
+*Kubernetes* menyediakan sebuah pengendali penerimaan (*PodPreset*) dimana, ketika diaktifkan, *Pod Preset* diterapkan kepada permintaan *pod* yang datang. Ketika sebuah penciptaan *pod* terjadi, sistem melakukan hal-hal berikut:
+
+1. Mengambil semua *PodPreset* yang tersedia untuk digunakan.
+2. Cek jika *label selector* dari salah satu *PodPreset* cocok dengan *label* pada *pod* yang sedang diciptakan.
+3. Usaha untuk menggabungkan berbagai resource didefinisikan oleh *PodPreset* ke dalam *Pod* yang sedang diciptakan.
```

Comment:
Nit: make resource utalic

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/5e8fb53a826b1119b77b923408a4f7a9e64e6415